core/silkworm/execution/evm.cpp

```diff
@@ -224,7 +225,22 @@ evmc::result EVM::call(const evmc_message& message) noexcept {
     return res;
 }
 
+evmc_address EVM::recipient_of_call_message(const evmc_message& message) noexcept {
```

Comment:
This is similar code for it in Aleth: https://github.com/ethereum/aleth/blob/master/libevm/ExtVMFace.cpp#L206-L208

I'm usually confused by this, but it seems that for the `CALLCODE` the answer is also `message.destination`.